src/test/scala/com/itechart/ny_accidents/utils/StringUtilsTest.scala

```diff
@@ -24,5 +24,18 @@ class StringUtilsTest extends FunSpec {
 
       assert(StringUtils.getLineMatchPercentage(firstString, secondString) == 0.0)
     }
+
+    it("should return wind string to double without dot") {
+      val testString = "1234.56"
+      val expectedValue = 123456
+
+      assert(StringUtils.windStringToDoubleParse(testString).contains(expectedValue))
```

Comment:
Why contains? "Equal" should work I guess?

Reply:
If I'm not mistaken, windStringToDoubleParse returns Option

Reply:
I got, thanks